### ISSUE TITLE
`geoCoordinates` where filter error

### ISSUE THREAD

**Issue by laura-ham** [semi-technologies/weaviate-javascript-client#6]
A GraphQL query like this with a where filter on geoCoordinates works:

```graphql
{
        operator: "WithinGeoRange",
        path: ["headquartersGeoLocation"],
        valueGeoRange: {
          geoCoordinates: {
            latitude: 51.51,    
            longitude: -0.09    
          },
          distance: {
            max: 2000 
          }
        }
      }
```

However, if this is used with the client like the following, the `geoCoordinates` object is parsed wrong, resulting in an error message:

```js
client.graphql
      .get()
      .withClassName('Article')
      .withFields('title InPublication{... on headquartersGeoLocation{latitude longitude}}')
      .withWhere({
        operator: "WithinGeoRange",
        path: ["headquartersGeoLocation"],
        valueGeoRange: {
          geoCoordinates: {
            latitude: 51.51,    
            longitude: -0.09    
          },
          distance: {
            max: 2000 
          }
        }
      })
      .do()
      .then(res => {
        console.log(JSON.stringify(res))
      })
      .catch(err => {
        console.error(err)
      });
```

error message:
```
{
  "errors": [
    {
      "locations": [
        {
          "column": 67,
          "line": 1
        }
      ],
      "message": "Syntax Error GraphQL request (1:67) Expected Name, found String \"geoCoordinates\"\n\n1: {Get{Things{Article(where:{operator:WithinGeoRange,valueGeoRange:{\"geoCoordinates\":{\"latitude\":51.51,\"longitude\":-0.09},\"distance\":{\"max\":2000}},path:[\"headquartersGeoLocation\"]}){title InPublication{... on headquartersGeoLocation{latitude longitude}}}}}}\n                                                                     ^\n",
      "path": null
    }
  ],
  "highlightQuery": "{Get{Things{Article(where:{operator:WithinGeoRange,valueGeoRange:{\"geoCoordinates\":{\"latitude\":51.51,\"longitude\":-0.09},\"distance\":{\"max\":2000}},path:[\"headquartersGeoLocation\"]}){title InPublication{... on headquartersGeoLocation{latitude longitude}}}}}}\n                                                                   ~~~~~~~~\n"
}
```

**Comment by etiennedi** [comment]
Should be fixed/implemented in v1.0.0-rc4. Keeping this issues open until I have feedback that it works as desired.

**Comment by etiennedi** [comment]
Confirmed working by @laura-ham. Closing.
